app/decorators/decidim/users/admin/users_controller_decorator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+#
+# This decorator adds the capability to the controller to query users
+# filtering by User role `department_admin`.
+#
+Decidim::Admin::UsersController.class_eval do
+  alias_method :original_method, :collection
+
+  private
+
+  def collection
```

Comment:
why do we need to filter department_admin roles?

Reply:
Ok I was wrong. A `department_admin` must only be able to "see" users from it's own department.